app/src/main/res/values/strings.xml

```diff
@@ -3,6 +3,7 @@
     <string name="url_privacy_policy">https://adeekshith.github.io/watomatic/#/privacy-policy.md</string>
     <string name="adeekshith_twitter_handle" translatable="false">\@adeekshith</string>
     <string name="url_adeekshith_twitter" translatable="false">https://twitter.com/adeekshith</string>
+    <string name="contributers_will_appear_here" translatable="false">contributers will appear here</string>
```

Comment:
Hi @aniket691 , can we remove this placeholder please?
We can show a generic "Loading" animation or text later but we may want to remove this for now as all the language translators will get notified of this and "contributors will appear here" without context does not make much sense for them.

Also, `translatable="false"` is only used for things like URLs which should not be translated to other languages :)
Thanks for this awesome contribution! This is great help!

Reply:
removed